lib/screens/routes/route.ex

```diff
@@ -62,6 +62,7 @@ defmodule Screens.Routes.Route do
   end
 
   @doc "Fetches routes that serve the given stop."
+  @callback serving_stop(Stop.id()) :: {:ok, [t()]} | :error
```

Comment:
I'm surprised this actually works, though it feels vaguely like a subversion of the Mox philosophy ("No ad-hoc mocks. You can only create mocks based on behaviours"). I guess you could say it's an "inline behavior"? Not an objection, just a "huh, that's interesting".

Experimentally, it seems to be sufficient to _only_ have the `@callback` — if you remove the duplicated `@spec`, Dialyzer still throws an error if you e.g. assign the return value to a pattern that can never match.

Reply:
Our dependency on the V3 API makes testing cumbersome. Probably a nicer solution would be to use use an adapter pattern for the V3 API.

For now I'll remove the redundant `@spec` attribute here.